.travis.yml

```diff
@@ -1,12 +1,30 @@
 language: php
+
+sudo: false
```

Comment:
@jpignata 

This sends builds to container-based infrastructure, see [Using container-based infrastructure](https://docs.travis-ci.com/user/workers/container-based-infrastructure):

>Jobs running on container-based infrastructure:

>1. start up faster
>2. allow the use of [caches](https://docs.travis-ci.com/user/caching) for public repositories
>3. disallow the use of sudo, setuid and setgid executables